Tests/SentryTests/SentryViewHierarchyProviderTests.swift

```diff
@@ -175,7 +169,7 @@ class SentryViewHierarchyProviderTests: XCTestCase {
     }
 
     func test_invalidSerialization() {
-        let sut = TestSentryViewHierarchyProvider()
+        let sut = TestSentryViewHierarchyProvider(dispatchQueueWrapper: SentryDispatchQueueWrapper(), sentryUIApplication: fixture.uiApplication)
```

Comment:
`l`: Could be moved to a `fixture.getSut()` helper method, like we do in other tests.
Not strong opinion here, so feel free to proceed at your discretion.

Reply:
Nei, I don't like the fixtures. Test code can be repetitive and stupid. I'm going to keep it. But thanks for the hint.